sds/Cargo.toml

```diff
@@ -28,8 +28,8 @@ regex-syntax = "0.7.5"
 serde = { version = "1.0", features = ["derive"] }
 serde_with = "3.6.1"
 thiserror = "1.0.58"
-metrics = "0.22.3"
-metrics-util = "0.16.3"
+metrics = "0.23.0"
```

Comment:
Right. [Vector is currently only on 0.23.0](https://github.com/vectordotdev/vector/blob/master/Cargo.toml#L142) so this is to match. I could push through a change there as well since it is behind a version as you note.